SQL/tgstation_schema.sql

```diff
@@ -532,6 +532,8 @@ CREATE TABLE `achievement_metadata` (
 	`achievement_key` VARCHAR(32) NOT NULL,
 	`achievement_version` SMALLINT UNSIGNED NOT NULL DEFAULT 0,
 	`achievement_type` enum('achievement','score','award') NULL DEFAULT NULL,
+	`achievement_name` VARCHAR(2048) NULL DEFAULT NULL,
+	`achievement_description` VARCHAR(2048) NULL DEFAULT NULL,

```

Comment:
I don't think these need to be 2048
The current longest achievement name is 26 bytes `Your Life Before Your Eyes` or even 36 bytes `My Watchlist Status is Not Important` from #48094
I'd suggest 64 since we also don't want massively long names, I imagine that'll mess with displaying them, but longer is fine. Anything less than 768 is fine so the record is inline on the table.
Same for description, I think allowing 2MiB of text is excessive. The longest description in #48094 is 195 bytes so maybe 255, 200, 512? idk
Similarly anything less than 768 is fine. More than that isn't a problem but it's a good demarkation point and someone shouldn't really be making a description that long anyway.

Reply:
Does mysql not have a proper TEXT column? who the fuck uses limited length varchar columns in 2019?

Reply:
I assumed we're not using it due to some compatibility requirement, but lemme check.

Reply:
Mysql does have TEXT types.
On InnoDB small varchars are quicker to retrieve since they can be stored in memory, up to a point take less storage and can be indexed.
There's no particular advantages to using TEXT with short strings.

Reply:
shoulda used postgresql